Software/Python/easygopigo3.py

```diff
@@ -476,6 +476,21 @@ def reset_encoders(self):
         self.offset_motor_encoder(self.MOTOR_LEFT,self.get_motor_encoder(self.MOTOR_LEFT))
         self.offset_motor_encoder(self.MOTOR_RIGHT,self.get_motor_encoder(self.MOTOR_RIGHT))
 
+    def read_encoders(self):
```

Comment:
I didn't think this was needed as it's pretty much the same as reaching down into the gopigo3 class, but I have no issue if Robert thinks he needs it. I would shy away from the word "encoder" as newcomers do not know what they are.  read_motor_position()  would be my preference.